tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -127,13 +147,14 @@ class OpenSSLServ(TLSProgram):
     }
 
     def cmd(self):
+        super().cmd()
         ret = ['$O_NEXT_SRV_NO_CERT']
-        for _, cert, key in map(lambda sig_alg: CERTIFICATES[sig_alg], self._sig_algs):
+        for _, cert, key in map(lambda sig_alg: CERTIFICATES[sig_alg], self._cert_sig_algs):
             ret += ['-cert {cert} -key {key}'.format(cert=cert, key=key)]
         ret += ['-accept $SRV_PORT']
         ciphersuites = ','.join(self._ciphers)
-        signature_algorithms = ','.join(self._sig_algs)
-        named_groups = ','.join(
+        signature_algorithms = ':'.join(self._sig_algs)
```

Comment:
Why change the character from ',' to ':'?

Reply:
`,` is wrong , the separate character is `:`